src/wallet/sendTransactionWalletService.ts

```diff
@@ -52,13 +52,13 @@ class SendTransactionWalletService extends EventEmitter implements ISendTransact
   private changeAddress: string | null;
 
   // Transaction object to be used after it's already prepared
-  private transaction: Transaction | null;
+  public transaction: Transaction | null;
 
   // MineTransaction object
   private mineTransaction: MineTransaction | null;
 
   // PIN to load the seed from memory
-  private pin: string | null;
+  public pin: string | null;
```

Comment:
We access `transaction` and `pin` in the RPC handler methods (which we should stop) but for now we need to make them public.

Reply:
Since this is security-related, should we create an issue in the RPC Lib to make sure this is fixed?

Reply:
I added a new commit to add some new parameters to util methods and now I don't need to make pin public anymore.